src/libraries/Microsoft.PowerFx.Core/Texl/Builtins/Hex2Dec.cs

```diff
@@ -55,5 +59,33 @@ public override string GetUniqueTexlRuntimeName(bool isPrefetching = false)
         {
             return GetUniqueTexlRuntimeName(suffix: "_T");
         }
+
+        public override bool CheckTypes(CheckTypesContext context, TexlNode[] args, DType[] argTypes, IErrorContainer errors, out DType returnType, out Dictionary<TexlNode, DType> nodeToCoercedTypeMap)
+        {
+            Contracts.AssertValue(args);
+            Contracts.AssertAllValues(args);
+            Contracts.AssertValue(argTypes);
+            Contracts.Assert(args.Length == argTypes.Length);
+            Contracts.Assert(args.Length == 1);
+            Contracts.AssertValue(errors);
+            Contracts.Assert(MinArity <= args.Length && args.Length <= MaxArity);
+
+            var fValid = base.CheckTypes(context, args, argTypes, errors, out returnType, out nodeToCoercedTypeMap);
+            Contracts.Assert(returnType.IsTable);
+            Contracts.Assert(!fValid || returnType.IsColumn);
+            var type = argTypes[0];
+            var arg = args[0];
+            if (type.IsTable)
```

Comment:
>if (type.IsTable) [](http://example.com/codeflow?start=12&length=17)

I think you don't need this If else, `base.CheckTypes` would have already validated this.